lib/kube/proxy/remotecommand_websocket.go

```diff
@@ -37,6 +37,7 @@ const (
 	preV4Base64WebsocketProtocol = wsstream.Base64ChannelWebSocketProtocol
 	v4BinaryWebsocketProtocol    = "v4." + wsstream.ChannelWebSocketProtocol
 	v4Base64WebsocketProtocol    = "v4." + wsstream.Base64ChannelWebSocketProtocol
+	v5BinaryWebsocketProtocol    = "v5." + wsstream.ChannelWebSocketProtocol
```

Comment:
```suggestion
	v5BinaryWebsocketProtocol    = remotecommand.StreamProtocolV5Name
```

Reply:
Drop 
```go

	stdinChannel = iota
	stdoutChannel
	stderrChannel
	errorChannel
resizeChannel

```

And use `remotecommand.StreamStdIn` constants instead, now that they are defined in kubernetes itself.